OpenRA.Mods.RA/AI/HackyAI.cs

```diff
@@ -310,7 +310,7 @@ public bool HasMinimumProc()
 		// For mods like RA (number of building must match the number of aircraft)
 		bool HasAdequateAirUnits(ActorInfo actorInfo)
```

Comment:
We should rename this to 'HasEnoughAirfields', or if you prefer - 'HasEnoughAircraftReloadingBuildings' :D